packages/react-components/react-overflow/src/useOverflowContainer.ts

```diff
@@ -78,6 +79,20 @@ export const useOverflowContainer = <TElement extends HTMLElement>(
     [overflowManager],
   );
 
+  const registerDivider = React.useCallback(
+    (divider: OverflowDivider) => {
+      const el = divider.element;
+      overflowManager?.addDivider(divider);
+      el && el.setAttribute(DATA_OVERFLOW_DIVIDER, '');
+
+      return () => {
+        divider.groupId && overflowManager?.removeDivider(divider.groupId);
+        el.removeAttribute(DATA_OVERFLOW_DIVIDER);
```

Comment:
if you have time to do this I would appreciate it, but I won't block on this: would make sense to set all these data attributes in the priority-overflow package actually, since they don't actually have anything to do with React

Reply:
Can you create an issue for this and resolve this thread?

Reply:
https://github.com/microsoft/fluentui/issues/28236